### PR TITLE
[Intel MKL] MKL DNN: cleaning mkl softamx kernel

### PR DESCRIPTION
**Pull request by ashraf-bhuiyan**
[Intel MKL] Cleaned up Intel MKL ML related code. The intention is to stop supporting MKL ML (closed source library) code and have the MKL DNN code (Open sourced library) to be the default code path.

### CODE REVIEW

**Comment by ashraf-bhuiyan** [comment]
Hi, if possible, can you please take a look at this PR. It is a very small change to one file to clean up the MKL ML code.

**Comment by ashraf-bhuiyan** [comment]
Hi, if possible, can you please take a look at this PR. It is a very small change to one file to clean up the MKL ML code.

**Comment by xiejw** [comment]
sorry, did not notice it was assigned to me. I am not familiar with this code. Can you find another reviewer? thanks

File: tensorflow/core/kernels/mkl_softmax_op.cc [hunk 1]
```diff
@@ -64,12 +62,13 @@ class MklSoftmaxOp : public OpKernel {
...
       memory::format layout_type;
-      // In MKL, data format passed to mkl softmax op depends on dimension of the input tensor.
-      // Here "x" data format in MKL is used for 1 dim tensor, "nc" for 2 dim tensor, 
-      // "tnc" for 3 dim tensor, "nchw" for 4 dim tensor, and "ncdhw" for 5 dim tensor.
+      // In MKL, data format passed to mkl softmax op depends on dimension of
+      // the input tensor. "x" data format in MKL is used for 1 dim tensor,
+      // "nc" for 2 dim tensor, "tnc" for 3 dim tensor, "nchw" for 4 dim tensor,
+      // and "ncdhw" for 5 dim tensor.
       // Each of the simbols has the following meaning:
       // n = batch, c = channels, t = sequence lenght, h = height,
```
Comment (tatianashp) [inline tensorflow/core/kernels/mkl_softmax_op.cc]: typo

Comment (tatianashp) [reply tensorflow/core/kernels/mkl_softmax_op.cc]: 'sequence lenght'

File: tensorflow/core/kernels/mkl_softmax_op.cc [hunk 2]
```diff
@@ -87,9 +86,11 @@ class MklSoftmaxOp : public OpKernel {
           layout_type = memory::format::ncdhw;
           break;
         default:
-          OP_REQUIRES_OK(context, errors::Aborted("Input dims must be <= 5 and >=1"));
+          OP_REQUIRES_OK(context, errors::Aborted(
+              "Input dims must be <= 5 and >=1"));
```
Comment (tatianashp) [inline tensorflow/core/kernels/mkl_softmax_op.cc]: Please place on the same line if it fits into 80 characters.

**Review by tatianashp — CHANGES_REQUESTED** [review]
The code looks good. A few nits on the comments.

**Comment by ashraf-bhuiyan** [comment]
I will wait a few days to commit this PR, since another PR also changing the file.

**Comment by ymodak** [comment]
@ashraf-bhuiyan Can you please resolve the conflicts? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by ashraf-bhuiyan** [comment]
It will be worked on. Need to fix a conflicted file.

**Comment by ashraf-bhuiyan** [comment]
I merged the softmax file with the master, and removed the conflict, and thus, it seems lots of changes. But, it has actually fewer changes comparing to master, about 6 to 7 lines.

**Comment by ashraf-bhuiyan** [comment]
@tatianashp if possible please have a look in this PR. Only few line changes in this PR.

**Review by tatianashp — APPROVED** [review]

**Comment by tatianashp** [comment]
Looks good. Thanks for fixing the typos.

**Comment by rthadur** [comment]
code has been merged to master.
